pkg/migration/controllers/migration.go

```diff
@@ -213,10 +214,9 @@ func (m *MigrationController) migrateVolumes(migration *storkv1.Migration) error
 		// Now check if there is any failure or success
 		// TODO: On failure of one volume cancel other migrations?
 		for _, vInfo := range volumeInfos {
-			// Return if we have any volume migrations still in progress
 			if vInfo.Status == storkv1.MigrationStatusInProgress {
 				logrus.Infof("Volume Migration still in progress: %v", migration.Name)
```

Comment:
```suggestion
				logrus.Infof("Volume %v migration still in progress", vInfo.Volume)
```

Reply:
Above will give more use info as it prints each volume instead of repeating the same log with the migration name for multiple vols.

Reply:
Yeah should have been volumename.